### PR TITLE
Dev -> master for v1.3.0

### PR DESCRIPTION
**Pull request by pinin4fjords**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by WackerO** [comment]
Hey Jonathan, LGTM, but I just noticed while reviewing that I forgot to remove the shinyapp error for maxquant datasets. Can we maybe merge #179 into dev and add to this PR before merging so that this is integrated into the release?

**Review by WackerO — APPROVED** [review]

File: nextflow.config [hunk 1]
```diff
@@ -39,19 +40,32 @@ params {
...
+    affy_bgversion           = 2
     affy_destructive         = false
     affy_cdfname             = null
     affy_rm_mask             = false
     affy_rm_outliers         = false
     affy_rm_extra            = false
     affy_build_annotation    = true
 
+    // Proteus-specific options
+    proteus_measurecol_prefix = 'LFQ intensity '
```
Comment (sateeshperi) [inline nextflow.config:62]:
```suggestion
    proteus_measurecol_prefix = 'LFQ intensity'
```

Comment (sateeshperi) [reply nextflow.config:62]: or is this intentional ?

Comment (WackerO) [reply nextflow.config:62]: The space is necessary :)

**Pull request merged**